Sts1CobcSw/Hal/Spi.hpp

```diff
@@ -10,6 +10,25 @@
 
 namespace sts1cobcsw::hal
 {
+class Spi
+{
+private:
+    RODOS::HAL_SPI spi;
```

Comment:
Rename `spi` to `spi_` since it is a private member variable. Same for `transferEnd`. Clang-tidy should have given you a warning about that.